src/theme.ts

```diff
@@ -121,5 +121,12 @@ const themeOverride = createTheme({
   },
   breakpoints,
 });
-
+export const CHART_COLORS = [
+  accentCyan[4],

```

Comment:
OK so this is great, but if we are already using the theme colors we dont really need the chart colors. 

Sorry if i was not understandable in my last pr